macantine/extract_open_data.py

```diff
@@ -48,8 +48,10 @@ def map_communes_infos():
         communes = response_commune.json()
         for commune in communes:
             commune_details[commune["code"]] = {}
-            commune_details[commune["code"]]["department"] = commune["codeDepartement"]
-            commune_details[commune["code"]]["region"] = commune["codeRegion"]
+            if "department" in commune.keys():
```

Comment:
C'est quoi le structure de données `commune` car ici tu vérifie pour `commune["department"]` mais la ligne suivante tu utilises `commune["codeDepartment"]`. C'est le même pattern pour `region`. 

Aussi pour les `dict`s on peut faire `if "keyname" in my_dict`. Je sais pas par contre si `commune` est un `dict` ou non.

Est-ce que ça a du sens de faire `if "codeDepartment" in commune` ?